From bd0638d33b59ee581a5c36a040678f5f5da54512 Mon Sep 17 00:00:00 2001
From: Saurabh Dubey <sdubey@codeaurora.org>
Date: Wed, 28 Mar 2018 11:42:48 +0530
Subject: [PATCH] Gralloc2: Mark some gralloc bits as valid

Add GRALLOC_USAGE_PRIVATE_WFD and GRALLOC_USAGE_PRIVATE_10BIT_TP
to list of valid gralloc flags.

Change-Id: Ia73ccf0071b83b247635baadef0cb88452b6aab2
CRs-Fixed: 2205241
---
 libs/ui/Gralloc2.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libs/ui/Gralloc2.cpp b/libs/ui/Gralloc2.cpp
index d565706431..105d2cbfa5 100644
--- a/libs/ui/Gralloc2.cpp
+++ b/libs/ui/Gralloc2.cpp
@@ -43,7 +43,7 @@ uint64_t getValid10UsageBits() {
             bits = bits | bit;
         }
         // TODO(b/72323293, b/72703005): Remove these additional bits
-        bits = bits | (1 << 10) | (1 << 13);
+        bits = bits | (1 << 10) | (1 << 13) | (1 << 21) | (1 << 27);
 
         return bits;
     }();
-- 
2.17.1

